src/transformers/data/datasets/glue.py

```diff
@@ -8,12 +8,16 @@
 from torch.utils.data.dataset import Dataset
 
 from filelock import FileLock
+from transformers import (
+    BartTokenizer,
+    BartTokenizerFast,
+    PreTrainedTokenizer,
+    RobertaTokenizer,
+    RobertaTokenizerFast,
+    XLMRobertaTokenizer,
+    logging,
+)
 
-from ...tokenization_bart import BartTokenizer, BartTokenizerFast
-from ...tokenization_roberta import RobertaTokenizer, RobertaTokenizerFast
-from ...tokenization_utils import PreTrainedTokenizer
-from ...tokenization_xlm_roberta import XLMRobertaTokenizer
-from ...utils import logging
```

Comment:
Because this break when sentencepiece or tokenizers are not installed. Importing from the root will import the dummy object instead (which is fine for us)